IDL/maps.idl

```diff
@@ -0,0 +1,175 @@
+#include "helpers/basic_inner_types.idl"
+
+typedef short alias_array[2];
+typedef sequence<short> alias_sequence;
+typedef map<long, long> alias_map;
+
+struct MapShort
+{
+    map<long, short> var_map_short;
+};
+
+struct MapUShort
+{
+    map<long, unsigned short> var_map_ushort;
+};
+
+struct MapLong
+{
+    map<long, long> var_map_long;
+};
+
+struct MapULong
+{
+    map<long, unsigned long> var_map_ulong;
+};
+
+struct MapLongLong
+{
+    map<long, long long> var_map_longlong;
+};
+
+struct MapULongLong
+{
+    map<long, unsigned long long> var_map_ulonglong;
+};
+
+struct MapFloat
+{
+    map<long, float> var_map_float;
+};
+
+struct MapDouble
+{
+    map<long, double> var_map_double;
+};
+
+struct MapLongDouble
+{
+    map<long, long double> var_map_longdouble;
+};
+
+struct MapBoolean
+{
+    map<long, bool> var_map_boolean;
+};
+
+struct MapOctet
+{
+    map<long, octet> var_map_octet;
+};
+
+struct MapChar
+{
+    map<long, char> var_map_char;
+};
+
+struct MapWChar
+{
+    map<long, wchar> var_map_wchar;
+};
+
+struct MapString
+{
+    map<long, string> var_map_string;
+};
+
+struct MapWString
+{
+    map<long, wstring> var_map_wstring;
+};
+
+struct MapEnum
+{
+    map<long, MyEnum> var_map_enum;
+};
+
+struct MapBitMask
+{
+    map<long, MyBitMask> var_map_bitmask;
+};
+
+struct MapAlias
+{
+    map<long, MyAlias> var_map_alias;
+};
+
+struct MapShortArray
+{
+    map<long, alias_array> var_map_short_array;
+};
+
+struct MapSequence
+{
+    map<long, alias_sequence> var_map_sequence;
+};
+
+struct MapMap
+{
+    map<long, alias_map> var_map_map;
+};
+
+struct MapUnion
+{
+    map<long, MyUnion> var_map_union;
+};
+
+struct MapStructure
+{
+    map<long, MyStructure> var_map_structure;
+};
+
+struct MapBitset
+{
+    map<long, MyBitset> var_map_bitset;
+};
+    
+struct MapKeyShort
+{
+    std::map<short, short> var_map_short;

```

Comment:
```suggestion
    map<short, short> var_map_short;
```

These is repeated in the following struct definitions.

Reply:
My autocomplete loves to do that....
Removed in [45f5a04](https://github.com/eProsima/dds-types-test/pull/1/commits/45f5a04ea19d9a9d5c3320836c663922d0f039d9)